masterlist.yaml

```
@@ -22745,12 +22745,22 @@ plugins:
         itm: 26
 
   - name: 'The Great City of Dawnstar.esp'
-    url: [ 'https://www.nexusmods.com/skyrimspecialedition/mods/19491/' ]
-    msg: [ *alreadyInTGC ]
+    url: 
```

There's some excess whitespace:

```suggestion
    url:
```